code/controllers/subsystem/wardrobe.dm

```diff
@@ -209,7 +209,9 @@ SUBSYSTEM_DEF(wardrobe)
 
 /// Take an existing object, and insert it into our storage
 /// If we can't or won't take it, it's deleted. You do not own this object after passing it in
-/datum/controller/subsystem/wardrobe/proc/stash_object(atom/movable/object)
+/datum/controller/subsystem/wardrobe/proc/stash_object(obj/item/object)
+	if(object.item_flags & DO_NOT_WARDROBE)
+		return
```

Comment:
no, BAD, users who assume typesafety for no reason get the newspaper. these are only items because nothing else uses the stashing system, there is no reason to assume this will always be the case